javascript/0071-simplify-path.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Stack
+ * https://leetcode.com/problems/simplify-path/
+ * 
+ * Time O(n) | Space O(n)
+ * @param {string} path
+ * @return {string}
+ */
+var simplifyPath = function(path) {
+    let currunt = '';
+    let myStack = [];
+    path = '/' + path + '/';
+    for(let i = 0; i < path.length; i++) {
+        
+        console.log(myStack);
+        if(path[i] === '/') {
```

Comment:
Suggestion: Avoid nesting more than 3 times

Reply:
I don't see any other way. Can you kindly show me How can I avoid nested if statements and still get the logic right? Thank you :)

Reply:
use `continue;` when you need to exit early to avoid else